### PR TITLE
keybase: update to 6.4.0.

### PR DESCRIPTION
**Pull request by lee2sman**
Hi, as discussed in #48036 (and encourage by @classabbyamp  to begin to work through them) I am attempting to start resolving breaking changes in updating the template for the keybase client. Unfortunately, I have no prior experience programming in Go. 

When attempting to package this new version of the Keybase client I get the error

```
=> ERROR: keybase-6.4.0_1: do_build: 'go install -p "$XBPS_MAKEJOBS" -v -tags "${go_build_tags}" -ldflags "${go_ldflags}" ${go_package}' exited with 1
=> ERROR:   in do_build() at common/build-style/go.sh:59
```

I think this is a permissions error but I'm not sure how to get more useful info from this error.

Any help majorly appreciated.

### CODE REVIEW

**Comment by lee2sman** [comment]
Thanks for your help @pondichys this template compiled and built and successfully runs on x86_64. I believe my next step will be to put attention to the ```kbfs``` and ```keybase-desktop``` templates!

File: srcpkgs/keybase/template [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+go_import_path="github.com/keybase/client/go"
+go_package="${go_import_path}/keybase
+${go_import_path}/kbfs/kbfsfuse
+${go_import_path}/kbfs/kbfsgit/git-remote-keybase
+${go_import_path}/kbfs/kbfstool ${go_import_path}/kbfs/redirector"
 go_build_tags="production"
 depends="gnupg>=2"
 short_desc="Client for keybase.io"
-maintainer="Toyam Cox <Vaelatern@voidlinux.org>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (classabbyamp) [inline srcpkgs/keybase/template:15]: @Vaelatern

File: srcpkgs/keybase/template [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
-go_package="${go_import_path}/go/keybase
-${go_import_path}/go/kbfs/kbfsfuse
-${go_import_path}/go/kbfs/kbfsgit/git-remote-keybase
-${go_import_path}/go/kbfs/kbfstool ${go_import_path}/go/kbfs/redirector"
+build_wrksrc=go
+go_import_path="github.com/keybase/client/go"
+go_package="${go_import_path}/keybase
+${go_import_path}/kbfs/kbfsfuse
+${go_import_path}/kbfs/kbfsgit/git-remote-keybase
+${go_import_path}/kbfs/kbfstool ${go_import_path}/kbfs/redirector"
```
Comment (Vaelatern) [inline srcpkgs/keybase/template:11]: I'm not sure why these changes were necessary

File: srcpkgs/keybase/template (hunk 1 above)
Comment (Vaelatern) [reply srcpkgs/keybase/template:15]: I don't use it anymore but I don't think I'm ready to orphan it. Just ready to not have merged updates for..... 3 years.

File: srcpkgs/keybase/template (hunk 2 above)
Comment (lee2sman) [reply srcpkgs/keybase/template:11]: According to @pondichys this is the go import path in go.mod file

Comment (lee2sman) [reply srcpkgs/keybase/template:11]: Sorry, to clarify: the go.mod file import path appends /go sub-directory.

Comment (Vaelatern) [reply srcpkgs/keybase/template:11]: Ah sure.

**Comment by Piraty** [comment]
for next time: squash them commits
